src/firmware_logger_device.h

```diff
@@ -1,66 +1,85 @@
 // License: Apache 2.0. See LICENSE file in root directory.
-// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
```

Comment:
I don't think you need to change the date when you modify a file

Reply:
It is something me and others have been doing as of lately. Trying to google if this is a good practice, I came across mixed opinions if this is necessary or even good to do this.
@Nir-Az, this is a legal issue, who should we contact about this?

Reply:
Normally I followed a role that a new file gets new year and modified not.
But I am not sure if any restrictions or best practice here.
I will check and update

Reply:
As instructed, edited files should show range of years. Fixing here to 2020-2024